### PR TITLE
Modified List (now Read) method, added find. delete and initialize method

### PR DESCRIPTION
**Pull request by ducampcarlos**

### CODE REVIEW

File: src/CRUD.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	    		for(T t: list) {
+	    			if(t.getId()==id) {
+	    				obj=t;
+	    			}
+	    		}
+	    	}else {
+	    		throw new NullPointerException("The list is empty");
+	    	}
+	    	return obj;
+	    }
```
Comment (AlejandroWoloch) [inline src/CRUD.java:47]: if the object isn't in the list it will return null, wouldn't be nice to throw an Exception there?

File: src/CRUD.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	    	list= new ArrayList<T>();
+	    }
+	    
+	    void delete(int id) {
+	    	T obj=find(id);
+	    	if(obj!=null) {
+	    		list.remove(obj);
+	    	}else {
+	    		throw new Exception("There's no element with this ID");
+	    	}
```
Comment (AlejandroWoloch) [inline src/CRUD.java]: if you change what I said about find method, then this should be a try-catch to re-throw the exception

File: src/CRUD.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+	    }
+	    
+	    void delete(int id) {
+	    	T obj=find(id);
+	    	list.remove(obj);
+	    }
+	    
+	    T find(int id) {
+	    	T obj=null;
+	    	if(!list.isEmpty()) {
```
Comment (rightonhana) [inline src/CRUD.java]: I think you need to catch the Exception with a try/catch like...
```
try {
    T obj = null
    if (!list.isEmpty()) {
        for (T t : list) {
            if (t.getId() == id; {
                obj = t;
            }
        }
    }
} catch (NullPointerException e) {
    // code
} catch (Exception e) {
    // code
}

return result;
```

But we can discuss this with @AlejandroWoloch

**Comment by AlejandroWoloch** [comment]
Looks okey to me

**Pull request merged**